src/main/java/fr/socrates/Main.java

```diff
@@ -72,9 +69,14 @@ public static void main(String[] args) {
             switch (choice) {
                 case ONE:
                     consolePrinter.print("Ajouter un candidat : ");
-                    boolean create = candidateService.add(createCandidate(scanner, consolePrinter));
-                    if(!create)
-                        consolePrinter.print("Le candidat a deja ete ajoute");
+                    boolean create = false;
+                    try {
+                        create = candidateService.add(createCandidate(scanner, consolePrinter));
+                    } catch (CandidatePersisteDataException e) {
```

Comment:
You can collapse the two catches : `catch(CandidatePersisteDataException | CandidateExistingException e)`

Reply:
ok done